woocommerce_fusion/woocommerce/doctype/woocommerce_server/woocommerce_server.py

```
@@ -142,10 +228,53 @@ def get_item_docfields(self, doctype: str) -> List[dict]:
+	def get_allowed_inbound_statuses(self) -> List[str]:
+		"""Return allowed inbound Woo status slugs for this server (default: ["processing"])."""
+		allowed = self.get("allowed_inbound_statuses")
+		if isinstance(allowed, list):
+			return allowed
+		try:
+			if isinstance(allowed, str) and allowed.strip():
+				data = frappe.parse_json(allowed)
+				if isinstance(data, list):
+					return data
+		except Exception:
+			pass
```

Ruff (0.13.1)

`try`-`except`-`pass` detected, consider logging the exception

(S110)